homeassistant/components/fitbit/const.py

```diff
@@ -0,0 +1,143 @@
+"""Constants for the Fitbit platform."""
+from __future__ import annotations
+
+import datetime
+from typing import Final
+
+from homeassistant.const import (
+    CONF_CLIENT_ID,
+    CONF_CLIENT_SECRET,
+    LENGTH_FEET,
+    MASS_KILOGRAMS,
+    MASS_MILLIGRAMS,
+    PERCENTAGE,
+    TIME_MILLISECONDS,
+    TIME_MINUTES,
+)
+
+_CONFIGURING: Final[dict] = {}
+
+ATTR_ACCESS_TOKEN: Final = "access_token"
+ATTR_REFRESH_TOKEN: Final = "refresh_token"
+ATTR_LAST_SAVED_AT: Final = "last_saved_at"
+
+CONF_MONITORED_RESOURCES: Final = "monitored_resources"
+CONF_CLOCK_FORMAT: Final = "clock_format"
+ATTRIBUTION: Final = "Data provided by Fitbit.com"
+
+FITBIT_AUTH_CALLBACK_PATH: Final = "/api/fitbit/callback"
+FITBIT_AUTH_START: Final = "/api/fitbit"
+FITBIT_CONFIG_FILE: Final = "fitbit.conf"
+FITBIT_DEFAULT_RESOURCES: Final = ["activities/steps"]
+
+SCAN_INTERVAL: Final = datetime.timedelta(minutes=30)
+
+DEFAULT_CONFIG: Final[dict[str, str]] = {
+    CONF_CLIENT_ID: "CLIENT_ID_HERE",
+    CONF_CLIENT_SECRET: "CLIENT_SECRET_HERE",
+}
+DEFAULT_CLOCK_FORMAT: Final = "24H"
+
+FITBIT_RESOURCES_LIST: Final[dict[str, tuple[str, str | None, str | None]]] = {
```

Comment:
It's actually more about refactoring but I'd use `NamedTuple` here. This way items can have names 🙂

Reply:
Not sure if refactoring should overload this PR :slightly_smiling_face: